src/vs/platform/sharedProcess/electron-main/sharedProcess.ts

```diff
@@ -241,6 +243,8 @@ export class SharedProcess extends Disposable implements ISharedProcess {
 			appRoot: this.environmentMainService.appRoot,
 			codeCachePath: this.environmentMainService.codeCachePath,
 			backupWorkspacesPath: this.environmentMainService.backupWorkspacesPath,
+			defaultProfile: this.userDataProfilesService.defaultProfile,
+			currentProfile: this.userDataProfilesService.currentProfile,
```

Comment:
Maybe more elegant to have a single profile property that provides access to default and current profile?